data/mods/Magiclysm/monsters/Orcs.json

```
@@ -0,0 +1,101 @@
+[
+  {
+    "id": "mon_orc_warrior",
+    "type": "MONSTER",
+    "name": { "str": "Orc Warrior" },
+    "description": "This green brute is a orc warrior. It stares at you menacingly, club in hand.",
+    "default_faction": "orcs",
+    "bodytype": "human",
+    "species": [ "ORCS" ],
+    "volume": "60 L",
+    "weight": "70 kg",
+    "hp": 92,
+    "speed": 110,
+    "material": [ "flesh" ],
+    "symbol": "o",
+    "color": "green",
+    "aggression": 100,
+    "morale": 110,
+    "melee_skill": 5,
+    "melee_dice": 6,
+    "melee_dice_sides": 4,
+    "dodge": 10,
```

```suggestion
    "dodge": 7,
```
Probably rather less; the archers might have a bit more than the basic warriors, but still not more than 7.